code-studio/src/js/initApp/initApp.js

```diff
@@ -19,6 +19,13 @@ window.apps = {
   // Loads the dependencies for the current app based on values in `appOptions`.
   // This function takes a callback which is called once dependencies are ready.
   load: loadApp,
+  chooseApp: function (selectorProperties, appOptionsList) {
```

Comment:
Please explain in a function-level comment what `selectorProperties` is and how you will use it to select the `appOptions`